client/db.go

```diff
@@ -374,8 +376,14 @@ func (db *DB) Run(b *Batch) error {
 //
 // TODO(pmattis): Allow transaction options to be specified.
 func (db *DB) Tx(retryable func(tx *Tx) error) error {
+	_, file, line, ok := runtime.Caller(1)
```

Comment:
`runtime.Caller(1)` takes 500ns on my machine. For comparison, constructing the debug name from the file and line number using `fmt.Sprintf` takes an additional 500ns. So the total debug name construction takes 1us.